tests/Feature/Departments/DepartmentsTest.php

```
@@ -41,29 +45,29 @@ public function it_can_create_department()
     }
 
     #[Test]
-    public function it_can_delete_department()
+    public function it_can_delete_department(): void
     {
         /* Arrange */
         $department = Department::factory()->create();
 
         /* Act */
         $this->assertNotNull(Department::where('external_id', $department->external_id)->first());
```

PHPStan (2.2.5)

[warning] Call to an undefined static method App\Models\Department::where().


(staticMethod.notFound)